transcom/mymove
# B-19722-INT USCG Customer must enter EMPLID

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19722)

## Summary

USCG customers must enter EMPLID on account creation. This value is editable when a move is in draft state.

### How to test

1. Create a new customer
2. Select Coast Guard as branch and enter EMPLID
3. Go back and forth between pages and try to break (change branches, enter letters, etc)
4. If not broken advance through profile and orders creation until you have a move in draft. Access your profile screen and edit your EMPLID and try to break
5. If not broken, create a new customer and confirm that EMPLID must be unique
